servers.json

```diff
@@ -1388,5 +1388,12 @@
         "addresses": [
             "betterpvp.club"
         ]
+    },
+    {
+        "id": "treepark",
+        "name": "TreePark",
+        "addresses": [
+            "play.treepark.games"
```

Comment:
Please go through your checklist again.

Reply:
The issue with the address should be fixed, I did another commit